ansible_collections/arista/avd/roles/eos_designs/templates/connected_endpoints/monitor-sessions.j2

```diff
@@ -0,0 +1,18 @@
+{# Leaf connected_endpoint monitor-sessions #}
+{% if monitor_sessions_data is arista.avd.defined %}
```

Comment:
this would always be defined, since you set it yourself in the main.j2 template. I suggest to change to `if monitor_sessions_data | length > 0`

Reply:
sorry, my bad. the list is defined but length could be 0, not "None".. so it cant caught by arista.avd.defined